src/components/bookings/Bookings.js

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from "react";
+import Search from "./Search.js";
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "../../data/fakeBookings.json";
+import Loading from "./Loading.js";
+import AddBookingModal from "./AddBookingModal.js";
+
+const Bookings = () => {
+  const [bookings, setBookings] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+  const [sorter, setSorter] = useState(false);
+
+  const fetchData = async () => {
+    try {
+      const res = await fetch("https://cyf-react.glitch.me/");
+      if (res.status >= 200 && res.status < 300) {
+        const data = await res.json();
+        setBookings(data);
+        setLoading(false);
+      } else {
+        setLoading(false);
+        setError(true);
+        throw new Error(res.status);
+      }
+    } catch (error) {
+      setLoading(false);
+      setError(true);
+      console.log(error);
+    }
+  };
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const search = searchVal => {
```

Comment:
`search` can probably be replaced here with a better variable name. What are you searching for exactly?